po/pl.po

```diff
@@ -213,11 +217,13 @@ msgstr ""
 #: data/resources/ui/login.ui:281
 msgid "By signing up,\nyou agree to the <a href=\"\">Terms of Service</a>."
 msgstr ""
+"Poprzez założenie konta,\n"
+"akceptujesz <a href=\"\">Zasady Korzystania z Usługi </a>."
```

Comment:
Should this whitespace be removed?
```suggestion
"akceptujesz <a href=\"\">Zasady Korzystania z Usługi</a>."
```
![Bildschirmfoto von 2021-11-26 21-59-05](https://user-images.githubusercontent.com/3630213/143643727-93709c68-004d-4677-8165-923aef443c54.png)
![Bildschirmfoto von 2021-11-26 21-58-04](https://user-images.githubusercontent.com/3630213/143643714-6cea07e1-05cd-4434-882f-16474a43948b.png)

Reply:
I fixed that from weblate.